src/App.js

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
+import { Provider } from 'react-redux';
 import {
   BrowserRouter as Router,
   Routes,
   Route,
 } from 'react-router-dom';
+import store from './redux/configureStore';
 import Books from './components/Books';
 import Categories from './components/Categories';
 import Nav from './components/Nav';
 import './App.css';
 
 const App = () => (
```

Comment:
- [ ] Good job implementing some of the changes suggested by the previous reviewer however, you should follow the previous reviewer instructions in the provided image ☝🏼, kindly make sure your app looks exactly as the provided design.

|Your app|Zeplin design|
|-|-|
|![Screenshot_20221119_173544](https://user-images.githubusercontent.com/25227022/202861654-50ba32b0-fd4f-40ba-a122-ecec072016db.png)|![](https://user-images.githubusercontent.com/26767383/202846956-3e185b59-e996-4bfb-b272-5be644e09871.png)|

Reply:
@Ismailco Hello, Can you please clarify on what exactly I should correct on my design. Because I implemented all the changes requested for the other reviewer. I only left out the optional changes to submit quickly and meet deadlines. My design was made to be mobile responsive. That's the only reason the input section is made that one. And it was indicated as an optional change. Thank you

Reply:
@StarMindz, as the reviewer mentioned you need to implement the changes from the photo attached 
![Screenshot_20221119_184706](https://user-images.githubusercontent.com/25227022/202864451-2f50dca3-0577-48de-a95a-608919919f75.png)
![Screenshot_20221119_184729](https://user-images.githubusercontent.com/25227022/202864452-bec2f34f-a3a6-40bb-80f4-435c4a461207.png)